### PR TITLE
Enrich the Creating an Access Control List (ACL) rule doc

### PR DESCRIPTION
**Pull request by vasilii-b**
## Purpose of this pull request

This pull request (PR) adds several improvements to the "Creating an Access Control List (ACL) rule" page (namings, grammar), including the example of limiting the layout block rendering based on the ACL Resource.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.html

## Links to Magento source code

-  https://github.com/magento/magento2/blob/2.3.4/lib/internal/Magento/Framework/View/Layout/etc/elements.xsd#L209

whatsnew
Added the 'Content restrictions for admin users' section to the [Creating an Access Control List (ACL) rule](https://devdocs.magento.com/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.html) topic.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md [hunk 1]
```diff
@@ -114,6 +118,54 @@ protected function _isAllowed()
...
 If the user doesn't have permission, the action page displays an "Access Denied" message.
 
+### Page content partial restriction for admin users
+
+With ACL it is also possible to [render layout blocks dynamically]({{ page.baseurl }}/frontend-dev-guide/layouts/xml-manage.html#ref_config_block) on the page.
+
+It is enough to set the block's value for `aclResource` attribute:
+
+```xml
+<block class="Vendor\MyModule\Block\Type" name="block.example" aclResource="Vendor_MyModule::view_additional">
```
Comment (rogyar) [inline src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md]: Just a minor edit if we are talking about admin panel 

```suggestion
<block class="Vendor\MyModule\Block\Adminhtml\Type" name="block.example" aclResource="Vendor_MyModule::view_additional">
```

**Comment by vasilii-b** [comment]
Hi @rogyar,
Thanks for the suggestions. Those have been applied.
Thank you!

File: src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md [hunk 2]
```diff
@@ -66,19 +69,20 @@ In your module, create the `etc/adminhtml/menu.xml` file. This file defines a me
...
 | --------- | ----------- |
-| `id` | Unique string and should be in this format: `Vendor_ModuleName::resourceName` |
-| `title` | Title which is display on menu bar|
+| `id` | Unique string. Should be in the format: `Vendor_ModuleName::resourceName` |
+| `title` | Title which is displayed in menu bar|
 | `module` | Module which containing the current menu |
-| `sortOrder` | Position in which menu to be disaplay |
+| `sortOrder` | Position in which menu to be displayed |
 | `parent` | The another menu which is parent of current menu |
 | `action` | Url of the page which needs to be display after click the menu. It should be in following format: `front_name/controller_path/action` |
```
Comment (rogyar) [inline src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md]:
```suggestion
| `action` | Url of the page which needs to be displayed after clicking the menu. It should be in following format: `front_name/controller_path/action` |
```

File: src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md [hunk 3]
```diff
@@ -114,6 +118,53 @@ protected function _isAllowed()
...
+
+It is enough to set the block's value for `aclResource` attribute:
+
+```xml
+<block class="Vendor\MyModule\Block\Adminhtml\Type" name="block.example" aclResource="Vendor_MyModule::view_additional">
+    <!-- ... -->
+</block>
+```
+
+The `view/adminhtml/layout/custommenu_view_index.xml` example file below contains two blocks that shows information to the end user. One of which is accessible only to users with ACL `Vendor_MyModule::view_additional` allowed.
```
Comment (rogyar) [inline src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md]:
```suggestion
The `view/adminhtml/layout/custommenu_view_index.xml` example file below contains two blocks that show information to the end-user. One of which is accessible only to users with ACL `Vendor_MyModule::view_additional` allowed.
```

**Comment by vasilii-b** [comment]
Thank you @rogyar!
The suggestions have been applied.

**Comment by dobooth** [comment]
running tests

**Pull request merged**